packages/product-download-page/partials/download-cards/index.tsx

```diff
@@ -91,7 +91,9 @@ function Cards({
           </div>
         )}
         <div className={hasPackageManager ? styles.card : styles.soloCard}>
-          <span className={styles.cardTitle}>Binary Download</span>
+          <span className={styles.cardTitle}>
+            {prettyOs(os)} Binary Download
+          </span>
```

Comment:
Fixes an issue where it was difficult to know if tab content was actually changing. Here's an example of the previous behaviour:

### Before

![make-tab-difference-more-clear](https://user-images.githubusercontent.com/4624598/129247337-854ac472-e39a-4bec-ae06-5bba5921f914.gif)

### After

![after-tab-difference](https://user-images.githubusercontent.com/4624598/129247576-e60d724f-7788-4668-978a-9c8ab7115dbf.gif)